ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committing/ProcessTransactionSubmission.scala

```diff
@@ -158,7 +158,10 @@ private[kvutils] class ProcessTransactionSubmission(
       case None =>
         reject(
           recordTime,
-          buildRejectionLogEntry(transactionEntry, RejectionReason.PartyNotKnownOnLedger))
+          buildRejectionLogEntry(
+            transactionEntry,
+            RejectionReason.PartyNotKnownOnLedger(
+              s"Party not known '${transactionEntry.submitter}'")))
```

Comment:
Could we say that it is the party on behalf of which we are submitting is unknown?

Reply:
Text changed to s"Submitting party '${transactionEntry.submitter}' not known" as advised below.